api/build.gradle

```diff
@@ -179,7 +177,7 @@ buildscript {    // Configuration for building
     }
   }
   dependencies {
-    classpath 'org.springframework.boot:spring-boot-gradle-plugin:1.5.3.RELEASE'
+    classpath 'org.springframework.boot:spring-boot-gradle-plugin:1.5.22.RELEASE'
```

Comment:
Can this use the SPRING_BOOT_VERSION variable?

Reply:
I was unable to get that to work.

Reply:
@jmthibault79 can you please file a low-priority ticket on getting that to work? I suspect judicious use of variables will be important to the dev tool cleanup campaign.

Reply:
Good news everyone.  I figured it out for #2564 .